### ISSUE TITLE
Token aliases causes an error on auto generation

### ISSUE THREAD

**Issue by ryo33** [jsinger67/parol#22]
I encountered this error:
```
    × Failed to generate user trait source!
    ╰─▶ Can't find type of argument ProductBegin
```

After debugging, I noticed that the generated expanded grammar does not have the `ProductBegin`.
The reason for this seems to be that I created aliases for the same token as follows:

```
BlockBegin: "\{";
ProductBegin: "\{";
ApplyBegin: "\{";
MapBegin: "\{";
```

Either of the following features would solve this problem.
- Rename aliases into a single name. (`ProductBegin`, `ApplyBegin`, and `MapBegin` are handled as `BlockBegin`.)
- Show an error on the checking phase if aliases exist.

**Comment by jsinger67** [comment]
Hi Ryo,
Thank you for reporting this bug.
I can confirm that this situation is currently not checked and therefore can produce errors during generation phase.
My preferred solution currently is to show an error message because this would lead to terminal conflicts.

**Comment by ryo33** [comment]
Thanks, it seems nice.

**Comment by jsinger67** [comment]
Fixed with commit 29ae3c2
Have fun 🎉
